addons/binding/org.openhab.binding.vera/src/main/java/org/openhab/binding/vera/controller/json/Sdata.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.vera.controller.json;
+
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * @author Dmitriy Ponomarev
+ */
+public class Sdata {
+    @SerializedName("full")
+    private String full;
+
+    @SerializedName("version")
+    private String version;
+
+    @SerializedName("model")
+    private String model;
+
+    @SerializedName("zwave_heal")
+    private String zwave_heal;
+
+    @SerializedName("temperature")
+    private String temperature;
+
+    @SerializedName("serial_number")

```

Comment:
Are all these `@SerializedName` annotations necessary in these POJOs? It looks like everywhere the default fields are used so they can all be omitted.